plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestFileSystemCache.java

```diff
@@ -50,6 +50,8 @@ public void testFileSystemCache()
 
         FileSystem fs5 = getFileSystem(environment, "user");
         assertNotSame(fs5, fs1);
+
+        environment.doAs("user", () -> FileSystem.getLocal(new Configuration(false)));
```

Comment:
I don't see `FileSystem.getLocal` anywhere in our codebase. 
Why it this a useful thing to test?

also, related, what is the end-user visible effect of the change?

Reply:
I encouter ClassCastException when integrating hadoop-huaweicloud which invoking` FileSystem.getLocal`.

Reply:
Can you share the entire stack trace ?